test/plugin/test_out_elasticsearch_data_stream.rb

```diff
@@ -221,6 +221,23 @@ def test_nonexistent_data_stream
     assert_equal "foo", driver(conf).instance.data_stream_name
   end
 
+  def test_placeholder
```

Comment:
Any chance to add testcases for time and custom record key placeholders?

Reply:
Sure,

Reply:
Currently, time placeholder testcase is missing. It is not resolved this issue. Thanks for your work! :+1:

Reply:
`data_stream_name 'foo_%Y%m%d'` should work.